src/angular-local-storage.js

```diff
@@ -311,9 +313,23 @@ angular
             }
             if (!!key) {
               var cookiePath = "; path=" + cookie.path;
-              if(cookie.domain){
+              if (cookie.domain) {
                 cookieDomain = "; domain=" + cookie.domain;
               }
+              /* Providing the secure parameter always takes precedence over config 
+               * (allows developer to mix and match secure + non-secure) */
+              if (isDefined(secure) && secure !== null) {
```

Comment:
If I take the code above, we lose the functionality of being able to explicitly override what config says.

At the moment - if config specifies `secure = true` I can call `cookie.set("key", "value", 3, false)` and it allows me to override what the config says (by going into the first "if" and obviously not going into the `if-else`).

If I take the code above and we specify secure as `false` we lose that functionality and will fall in to the `if-else` which will take the value from config. Which renders the secure parameter on `cookie.set` useless...

what do you think???